doc/en/user/source/installation/index.rst

```diff
@@ -17,3 +17,5 @@ There are many ways to install GeoServer on your system. This section will discu
    linux
    war
    upgrade
+
+.. note:: At this time we no longer provide a Windows Installer, because the benefics do not outstand the costs (it's basically a wrapper around the win_binary). However if you realy need the Windows installer, you can create one following `the instruction on this site <../../developer/win-installer.html>`.
```

Comment:
The wording is misleading, it's not the cost benefit per se, we need:
- A well secured Windows server managed by a trusted organization that can host the OSGeo certificate used to sign installers. This alone would reinstate the presence of the installer in releases.
- Someone to redo the installer from scratch, using Tomcat  instead of a reduced Jetty, to get a production worthy installer, instead of a demo oriented one. Not needed, but useful for those that actually want to run production on Windows.

Reply:
I was looking for a 'simple' an 'small' note, without a complete motivation.
Therefore I now replaced the 'specific' motivation by an 'open' motivation.

Or would you rather have this motivation written out? 
If so, it would be best to have it outside the 'note' section?

Reply:
"At this time we no longer provide a Windows Installer, due to lack of a secure Windows machine where the installer can be built and signed" would be better.

Reply:
Final update using your suggestion @aaime